sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -64,6 +64,30 @@ Only plain objects and arrays [are made deeply reactive](/#H4sIAAAAAAAAE42QwWrDM
 
 In non-runes mode, a `let` declaration is treated as reactive state if it is updated at some point. Unlike `$state(...)`, which works anywhere in your app, `let` only behaves this way at the top level of a component.
 
+## `$state.link`
+
+State declared with `$state.link` behaves like `$state`, with the key difference being that rather than take an initial value, the value passed into `$state.link` maintains and persistant one-way link. Updating the `$state.link` directly will temporarily override the value until the linked state next updates.
+
+```svelte
+<script>
+	let { parent_value = $bindable() } = $props();
+
+	// `child_value` is linked to `parent_value`
+	let child_value = $state.link(parent_value);
+
+	parent_value = 10;
+
+	console.log(child_value); // 10
+
+	child_value = 20;
+
+	console.log(child_value); // 20
+
+	// `parent_value` remains the same
+	console.log(parent_value); // 10
+</script>
+```
```

Comment:
Maybe something additional like "note that if you mutate the value, you're mutating the original value, unless you do something like `$state.snapshot` to clone it" here?

Also, can you copy that section over to `documentation/docs/03-runes/01-state.md`? That's where the docs for the final site live.

Reply:
Good point